nanoFirmwareFlasher.Library/FirmwarePackage.cs

```diff
@@ -342,7 +360,28 @@ internal async Task<ExitCodes> DownloadAndExtractAsync()
                             Console.ForegroundColor = ConsoleColor.White;
                         }
 
+                        //send app insight on successful download
+
+                        string insightConnectionString = _configuration["iConnectionString"];
+                        string optOut = Environment.GetEnvironmentVariable("DOTNET_CLI_TELEMETRY_OPTOUT");
```

Comment:
Can we rename it NANOFRAMEWORK_TELEMETRY_OPTOUT?

Reply:
Good catch, forgot to change this variable. 
Done.